packages/main/src/TableCellBase.ts

```diff
@@ -31,6 +32,17 @@ abstract class TableCellBase extends UI5Element {
 	@property({ type: Boolean })
 	_popin = false;
 
+	/**
+	 * Determines the horizontal alignment of table cells.
+	 * Note: All values valid for justify-content can be used not just the ones inside the enum.
```

Comment:
This is a bit hard to read in the WC pages. I found this in https://sap.github.io/ui5-webcomponents/nightly/components/List/: Note is bold , looks much better.